conf/modules/cv_colorfilter.xml

```diff
@@ -28,7 +28,7 @@
   <init fun="colorfilter_init()"/>
   <makefile>
     <file name="colorfilter.c"/>
-    <define name="USE_($COLORFILTER_CAMERA)"/>
+    <define name="USE_$(COLORFILTER_CAMERA)"/>
```

Comment:
Unless USE_VIDEO1 is explicitly defined in the video thread module, it doesn't initialize properly. My guess is that the preprocessor resolves video_thread.c before this line?